aim/db/migration/alembic_migrations/versions/8c6cbd136805_nested_parameter.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2016 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Create table for NestedParameter and VlanRange.
+
+Revision ID: 8c6cbd136805
+Revises: 3880e0a62e1f
+Create Date: 2020-02-21 19:02:47.560600
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8c6cbd136805'
+down_revision = '3880e0a62e1f'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table(
+        'aim_nested_parameter',
+        sa.Column('cluster_name', sa.String(64), nullable=False),
+        sa.Column('domain_name', sa.String(64)),
+        sa.Column('domain_type', sa.String(32), nullable=False),
+        sa.Column('domain_infra_vlan', sa.Integer, nullable=False),
+        sa.Column('domain_service_vlan', sa.Integer, nullable=False),
+        sa.Column('domain_node_vlan', sa.Integer, nullable=False),
+        sa.Column('epoch', sa.BigInteger(), nullable=False,
+                  server_default='0'),
+        sa.PrimaryKeyConstraint('cluster_name'))
```

Comment:
Can cluster names overlap across OpenStack projects (and therefore AIM tenants)? If so, we may want to add tenant name in here as a primary key, or create a UUID to use to make sure this is unique.

Reply:
Good point. Will add a project_id column and make it the primary key also along with the cluster name.